internal/runners/initialize/init.go

```diff
@@ -258,6 +258,8 @@ func verifyLangAndVersion(lang language.Language, version string) error {
 		return locale.NewInputError("err_init_language_not_found", "The selected language cannot be found")
 	}
 
+	version = strings.Replace(strings.Replace(version, ".x", "", 1), ".X", "", 1) // strip any wildcard
```

Comment:
We're just verifying the wildcarded version exists in the first place. e.g. 3.10.x will check that 3.10 is a valid version prefix.